titles-enterprise/getting-started/title.adoc

```diff
@@ -15,12 +15,13 @@ using a variety of available assets. You can also use {PLANNER} to find the opti
 include::_artifacts/snip-conscious-language.adoc[]
 
 // Assembly includes
+include::assemblies/assembly-introduction.adoc[]
 include::assemblies/assembly-getting-started-decision-services.adoc[]
 ifdef::PAM[]
 include::assemblies/assembly-getting-started-process-services.adoc[]
 include::assemblies/assembly-getting-started-case-management.adoc[]
 endif::[]
-include::assemblies/assembly-getting-started-business-optimizer.adoc[]
+include::assemblies/assembly-optaplanner-employee-rostering.adoc[]
```

Comment:
Generally I understand why do you change that, but I also see that in the documentation the paths are always point to optimizer like here: [assemblies/assembly-getting-started-business-optimizer.adoc](https://github.com/emmurphy1/kie-docs/pull/352/commits/d4aa0377bdd5a18aa43671af3ca15bfb9a835abe#diff-6b9e879c8928f609ba5a8cf93d86c2665c0593b2522c5cb03e1de79a89e8fb6b)